src/ResourceManager/ServiceBus/Commands.ServiceBus/Cmdlets/Queue/NewAzureServiceBusQueue.cs

```diff
@@ -68,6 +68,11 @@ public class NewAzureRmServiceBusQueue : AzureServiceBusCmdletBase
         [ValidateNotNullOrEmpty]
         public bool? DeadLetteringOnMessageExpiration { get; set; }
 
+        [Parameter(Mandatory = false, ValueFromPipelineByPropertyName = true, HelpMessage = "Enable Batched Operations - value that indicates whether server-side batched operations are enabled")]
+        [ValidateSet("TRUE", "FALSE", IgnoreCase = true)]
+        [ValidateNotNullOrEmpty]
+        public bool? EnableBatchedOperations { get; set; }
```

Comment:
@v-Ajnava `bool` parameters are strongly discouraged in PowerShell -- please change this to be a `SwitchParameter`

Reply:
implemented SwitchParameter